src/index.html

```diff
@@ -22,15 +22,15 @@
         <button type="button" title="Selection Mode" class="modeButton">
           <i class="fa fa-mouse-pointer" aria-hidden="true"></i>
         </button>
-        <button type="button" title="Move Mode" class="modeButton">
+        <button type="button" onclick= "dragMode()" title="Move Mode" class="modeButton">
           <i class="fa fa-arrows" aria-hidden="true"></i>

```

Comment:
lets change this icon to a hand maybe 
https://forkaweso.me/Fork-Awesome/icon/hand-paper-o/

Reply:
Oh I was wondering how this was done, very cool feature of HTML